docs/admin_guide.md

```diff
@@ -0,0 +1,145 @@
+   - [Integrated Toolforge Tools](#integrated-toolforge-tools)
+   - [Other Integrated APIs and Third-Party Dependencies](#other-integrated-apis-and-third-party-dependencies)
+2. [Monitoring and Logs](#monitoring-and-logs)
+   - [Toolforge](#toolforge)
+   - [Cloud VPS](#cloud-vps)
+3. [Troubleshooting](#troubleshooting)
+   - [Web Server Issues](#web-server-issues)
+   - [Database Issues](#database-issues)
+   - [Data Dumps and Recovery](#data-dumps-and-recovery)
+4. [More Resources](#more-resources)
+
+## Infrastructure Overview
+The **Program & Events Dashboard** is hosted within the **Wikimedia Cloud VPS** project [globaleducation](https://openstack-browser.toolforge.org/project/globaleducation), which provides the infrastructure for all servers, allowing the dashboard to run on virtual machines that are flexible and easily managed within Wikimedia Cloud. 
+
+The dashboard relies on several core servers and external tools to function. These components ensure that different tasks are isolated to avoid bottlenecks and improve system performance.
+
+### Servers
+The dashboard operates on a distributed server architecture to handle web requests, process background jobs, and store application data. Each server is dedicated to specific roles, minimizing competition for resources and improving reliability by isolating potential bottlenecks and failures.
+
+Below is a breakdown of the key servers and their roles within the infrastructure:
+
+1. **Web Server** 
+   - **`peony-web.globaleducation.eqiad1.wikimedia.cloud`**  
+     - Hosts the main web application and core Sidekiq processes using **RVM (Ruby Version Manager)**, **Phusion Passenger**, and **Apache**.
+     - **Capistrano** is used for deployments  
+     - Sidekiq processes hosted:  
+       - `sidekiq-default`: Manages frequently run tasks (e.g., adding courses to update queues).  
+       - `sidekiq-constant`: Handles transactional jobs (e.g., wiki edits, email notifications).  
+       - `sidekiq-daily`: Executes long-running daily update tasks.  
+
+2. **Sidekiq Servers**: These dedicated servers handle the other Sidekiq processes to isolate bottlenecks and failures:  
+   - **`peony-sidekiq.globaleducation.eqiad1.wikimedia.cloud`**: Hosts `sidekiq-short` for short-running course updates.  
+   - **`peony-sidekiq-medium.globaleducation.eqiad1.wikimedia.cloud`**: Hosts `sidekiq-medium` for typical course updates.  
+   - **`peony-sidekiq-3.globaleducation.eqiad1.wikimedia.cloud`**: Hosts `sidekiq-long` for long-running course updates with higher queue latency.  
+
+3. **Database Server**  
+   - **`peony-database.globaleducation.eqiad1.wikimedia.cloud`**: Stores program, user, and revision data. It supports the dashboard’s data queries and updates.
+
+4. **Redis Server**  
+   - **`p-and-e-dashboard-redis.globaleducation.eqiad1.wikimedia.cloud`**: Stores all task (job) details and is shared across all Sidekiq processes for task queuing and caching.
+
+### Integrated Toolforge Tools
```

Comment:
@ragesoss I added all the services you mentioned plus whocolor, wikidatadiffanalyzer and liftwing api. I can also remove them if they're unnecessary though.